website/docs/reference/network-view.mdx

```diff
@@ -8,76 +8,59 @@ title: Network View
 
-## Network overview
+## Connected Edges 
 
-The network overview is a diagram that shows the Unleash instance and known [applications](#applications) that have connected to it within the last five minutes. [Unknown applications](#unknown-applications) are not shown.
-
-Each application shown on the diagram has:
-- An application name
-- the average number of requests per second (_req/s_) that we have registered over the last five minutes.
-
-![Network overview chart](/img/network-overview.png)
-
-## Network traffic
-
-The network traffic diagram is a line chart that presents requests that have used the most network traffic over the last six hours, grouped by client and base URL for the request. For legibility, this chart only shows the ten groups that have caused the most traffic over the last six hours.
-
-Unleash aggregates requests by **client** (using application name) and **base URL**. Base URLs are batched together using the **first two** path segments following the `/api` part of the URL. In essence, that means:
-
-1. Separate requests by API: Admin API requests are separate from client API requests.
-2. Within each of these groups, group all requests by their next URL path segment. For instance: `/client/features` and `/client/features/feature-a` are grouped together, while `/client/register` and `/admin/features` are separate groups.
+:::note Availability
 
+[Enterprise](https://www.getunleash.io/pricing) | **Version**: `6.8+`
 
-![Network traffic chart](/img/network-traffic.png)
+:::
 
-## Data source
+The **Connected Edges** dashboard provides detailed observability metrics for Unleash Edge, helping teams efficiently manage all their connected Edge instances. 
+The dashboard offers a high-level view of all hosted Edge instances, with deeper insights into instance ID, region, CPU and memory usage, and upstream/downstream latency.
 
-:::info Prometheus and other sources
+This data helps you ensure that all instances are operational and quickly identify potential issues.
 
-The network view was written to be used with [Prometheus](https://prometheus.io/) and is therefore compatible with Prometheus' API.
+## Data Usage
 
-Other services that offer the same capabilities and the same API may work as substitutes, but we make no guarantees.
+The **Data Usage** dashboard provides an overview of your requests to Unleash within a specific month, along with aggregated data over the last 3, 6, or 12 months. This allows you to track and assess your data usage over time.
+You can also compare your usage in previous months with the current month.
 
-This section will refer to the external source as Prometheus for simplicity.
+## Configure a data source
 
-:::
+To populate the **Network** dashboard with data, you can integrate with [Prometheus](https://prometheus.io/), or similar tools.
```

Comment:
Is this Prometheus info up-to-date?

Reply:
Yes. This is still true. We use Victoria metrics, but that's Prometheus compatible.